area4/__init__.py

```diff
@@ -65,7 +65,20 @@ def divider(self, number) -> str:
                 return self.dividers[number].split("\n")[0]
             except IndexError:
                 raise ValueError('That divider doesn\'t exist!')
-
+    def logDivider(self, div, *stuff) -> str:
+        retstr = "";
+        if type(div) is int: div = self.dividers[div]
+        if len(stuff) == 1:
+            retstr = stuff[0]
```

Comment:
```suggestion

```

Reply:
I don't think we should. We want to return the only string given if you simply pass logDivider('', "Foo")

Reply:
What coffee is trying to say is that this is a simplified method.

Reply:
Oh, got it.